lib/base.rb

```diff
@@ -66,7 +66,11 @@ def output_testcases_to_file
   def write_to_file
     File.open(self.class::Conf.output_file, 'a') do |file|
       @tests.each do |test|
-        file.write(test.compile_template)
+        begin
+          file.write(test.compile_template)
+        rescue
+          # Cry deeply
+        end
       end
```

Comment:
Haha. But on the realz, I think you should move the logic in lines 68-73 to another method. It's that this `write_to_file` method is gaining too much responsibility.